src/imex.rs

```diff
@@ -781,6 +738,48 @@ where
     Ok(())
 }
 
+/// Exports the database to *dest*, encrypted using *passphrase*.
```

Comment:
Asfaik, the answer is no. the passphrase is only used for the database

Reply:
Though all data is encrypted on the wire, when transferred using `iroh`.

Reply:
I asked myself that before, but that decision was made when adding encryption to the DB asfaik, this PR just follows what was there in that regard.

Reply:
But then the whole archive needs to be encrypted, no?

Reply:
So yeah, the current storage encryption at rest has a lot of holes as explained, it's not really supported.  I went with encrypting here since that's what the other backup paths do and because we easily could so there's little to lose.

Reply:
> The idea was that we will eventually put the blobs inside the database

Good idea, maybe the only problem is impossibility of data deduplication if a user exports blobs outside of DC